jpo-asn-runtime/src/main/java/us/dot/its/jpo/asn/runtime/types/UnknownType.java

```diff
@@ -0,0 +1,9 @@
+package us.dot.its.jpo.asn.runtime.types;
+
+/**
+ * Placeholder for unknown types in the generated source.
+ * If classes of this type exist, it means the ASN.1 specification has unsupported features.
+ * @author Ivan Yourshaw
+  */
+public class UnknownType implements Asn1Type {

```

Comment:
question: is this just a stand-in for null?

Reply:
No, this is a marker for an error emitted by the compiler if it doesn't know how to deal with a type assignment.  ASN.1 has a NULL type that is used for various things, like as the value of some CHOICE types, but it has its own class here.

Reply:
Added more details to the Javadoc on this class